dashboard/config/scripts_json/csd3-2021.script_json

```diff
@@ -5090,7 +5090,7 @@
       "key": "b2ad1d43-99f9-40fa-9f88-d8b42f8c723e",
       "position": 7,
       "properties": {
-        "description": "Using the rubric, students should assess their own project before submitting it.\r\n\r\n\r\n&fa-desktop; Send students to Code Studio to complete their reflection on their attitudes toward computer science. Although their answers are anonymous, the aggregated data will be available to you once at least five students have completed the survey.",
```

Comment:
I don't know if we ever came to a consensus on whether to support `&fa-desktop`

Reply:
In this specific case, it looked like it was being used to display the "there's a tip" icon, which of course with the new system is automatically showing up when a "tip" is present.

I did also notice these fontawesome icons in a couple of other places and left them untouched, assuming that we'd get to them as part of the larger "special CB syntaxes" effort

Reply:
I think they were used to indicate that students should move to the computer at that point

Reply:
Ya fa-desktop was used to mark where there was a transition onto the computer. I think when we talked to the curriculum team they didn't seem attached to it

Reply:
I think if we didn't remove all the fa-desktops we should keep these ones and double check with curriculum before removing them all at once.

Reply:
ahhh, I definitely misunderstood what this one was being used for, then. I'll put it back for now